pkg/arvo/sys/hoon.hoon

```diff
@@ -12014,8 +12014,9 @@
   [p.gun (slum q.gat q.sam)]
 ::
 ++  slab                                                ::  test if contains
-  |=  {cog/@tas typ/type}
-  =(& -:(~(find ut typ) %free [cog ~]))
+  |=  {cog/@tas way/?(%read %rite %both) typ/type}
```

Comment:
I forbid %free because it violates the principle of information hiding.

Reply:
Ok.  A comment explaining that would be good, then.  While you're at it, I think you might as well modernize the syntax here.